presto-main/src/main/java/com/facebook/presto/FullConnectorSession.java

```diff
@@ -109,6 +114,12 @@ public long getStartTime()
     }
 
     @Override
+    public <T> T getSystemProperty(String name, Class<T> type)
+    {
+       return type.cast(systemProperties.get(name));
```

Comment:
Kamal, please investigate the issue more. Does it return null when setting the value though console or JDBC? `SessionPropertyManager` should return default value if no value is set by the user.
Current design is not clear because the default value is defined twice and it seems that the right place to define defaults is `SystemSessionProperties`